01-getting-started/getting-started.Rmd

```diff
@@ -5,4 +5,107 @@ output:
   html_notebook: 
     toc: true
     toc_float: true
----
\ No newline at end of file
+---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [About refine.bio](#about-refinebio)
+- [About how this tutorial book is structured](#about-how-this-tutorial-book-is-structured)
+- [What you need to install to run the examples](#what-you-need-to-install-to-run-the-examples)
+- [How to get the data](#how-to-get-the-data)
+- [How to use R Markdowns](#how-to-use-r-markdowns)
+- [An important note about file paths and Rmds](#an-important-note-about-file-paths-and-rmds)
+    - [Resources for learning R](#resources-for-learning-r)
+    - [More resources from the CCDL](#more-resources-from-the-ccdl)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## About refine.bio 
+
+[refinebio](https://www.refine.bio/about) is a collection of ready-to-use pre-processed normalized data! 
+Publicly available gene expression data is [uniformly processed](http://docs.refine.bio/en/latest/main_text.html#processing-information) and made available for easy download. 
+This tutorial has follow-along examples for use with refinebio downloads. 
+
+## About how this tutorial book is structured 
+
+This tutorial contains follow-along analysis examples for refinebio gene expression data.
+The analysis examples are organized by technology: ["microarray"](../02-microarray/intro-to-microarray.nb.html) or ["RNA-seq"](../03-rna-seq/intro-to-rna-seq.nb.html), in addition to an ["Advanced Topics"](../04-advanced-topics/intro-to-advanced-topics.nb.html) section. 
+Each analysis is self-contained and provides information with how to obtain the dataset used in the example from refinebio. 
+We encourage you to download the [`.Rmd`](http://rmarkdown.rstudio.com) and follow the "getting started" section in the example before diving into our analysis examples.
+
+**Each analysis contains:**   
+- A README that introduces you to the analyses, concepts, requirements, and workflows for that module.
+- An R Notebook which consists of:
+     - An R markdown (Rmd) file(s) that you can use in RStudio to run the analysis and contains it's own "getting started" section which describes how to download the example dataset from refinebio. 
+    - An nb.html file that is the resulting output of the Rmd file rendered as an HTML file.
+
```

Comment:
Maybe should add brief outline about what each getting started for each analysis contains. e.g. `Further learning resources about this analysis` and the rest of the sections. What information should a user expect to find for each analysis?